services/QuillLMS/app/queries/impact_metrics/schools_containing_certain_teachers_query.rb

```diff
@@ -2,13 +2,6 @@
 
 module ImpactMetrics
   class SchoolsContainingCertainTeachersQuery < ::QuillBigQuery::Query
```

Comment:
Not a high priority, but should we consider renaming this now that the logic for "certain teachers" is baked into the query itself instead of being passed in?  `SchoolsWithMinimumActivitySessions` or something?